src-ui/lib/state/Books.spec.ts

```diff
@@ -47,27 +47,24 @@ describe('books', () => {
   })
 
   describe('with several books preloaded', () => {
+    let duneMessiahId: string
+
     beforeEach(() => {
       books.add(duneMessiah)
```

Comment:
maybe `add` should return the resulting `id`; and/or the resulting `Book`? Then you don't have to do dead-reckoning later with `books.value[0]`...